tests/dummy/app/components/scroll-to/component.js

```diff
@@ -7,8 +7,8 @@ export default Component.extend({
 
   click() {
     next(() => {
-      let element = document.querySelector(this.get('href'));
-      let position = element.offsetTop + this.get('offset');
+      let element = document.querySelector(this.element.hash);
```

Comment:
I wasn't able to find out why.. but the `this.href` property isn't available on the component anymore. I was able to use `this.element.hash`

Maybe it's glimmer related? Something about the difference between html attributes, and component properties

Reply:
Its dummy app :) FIne for me